app/src/main/java/swati4star/createpdf/fragment/ImageToPdfFragment.java

```diff
@@ -674,4 +682,29 @@ void addMargins() {
                 })).build();
         materialDialog.show();
     }
+
+    private void addPageNumbers() {
+        MaterialDialog materialDialog = new MaterialDialog.Builder(getContext())
+                                            .title("Choose Page Number Style")
+                                            .customView(R.layout.add_pgnum_dialog, false)
+                                            .positiveText("OK")
+                                            .negativeText("Cancel")
```

Comment:
ok& cancel are already there in strings.xml. Use them

Also, we have utils file for dialog. YOu can use that directly...
https://github.com/Swati4star/Images-to-PDF/blob/master/app/src/main/java/swati4star/createpdf/util/DialogUtils.java#L65